### ISSUE TITLE
Installer doesn't let us choose installation path

### ISSUE THREAD

**Issue by CapoFantasma97** [atom/atom#20451]
## Summary
We are the users. We know better than a package installer where to put our software. There shouldn't even be the need to give motivations, but a main one is that there's no need to bloat up our SSD main drive with software that does not need the benefits of SSD.

## Motivation
Why does such a feature rich text and code editor still not have such an essential feature over the years? It's basic necessity and courtesy.

It blows my mind that by digging around, the only explaination given is an absurd "it would go against the silent one click design". How can one say that with a straight face? A "dumb" user will click on skip and move on. There, 2 click design.

## Describe alternatives you've considered
Put an option in the installer. Still feeling like your users are numnuts? Put a giant "install" button and a tiny "choose directory" below it. Or if you want it to be an "automatic one click installer" at all costs, put a 3 seconds countdown unless the user presses a key to choose the install directory. 

Literally anything you want but allow to choose the install location if one so desires.

## Additional context
Everything and my grandma's candy crush clone have an option to choose an installation path. Why does both the best and the worst of the software industry have this option but not Atom?

**Comment by j-bakewell** [comment]
i agree. i'm taking a course where the instructor recommends this editor. i'd like to use it, but i have a small ssd as a startup drive and a couple terabyte drives for everything else. it's 2020. i think this is the first program i've seen, maybe since the 80s, that doesn't offer a custom installation path. again, i'd prefer to use this program for my course, but am going to have to go with something else.

**Comment by LuNeder** [comment]
Super agree. Why doesn't a super custumizable and hackable program have such a simple option as choosing install directory?

**Comment by ThatXliner** [comment]
Um, is this Windows? I am on a Mac and when I click "download" I get a zip file that I can unzip anywhere I like.
